### DISCUSSION TITLE
Pyrogram errors.exceptions.bad_request_400.ChatAdminRequired: Telegram says: [400 CHAT_ADMIN_REQUIRED] - The method requires chat admin privileges (caused by "messages.SendMessage"

Help me with this please.🥺

### DISCUSSION THREAD

**Discussion by Noctophile** [anasty17/mirror-leech-telegram-bot#1750] [General]
Rizz

**Comment by anasty17** [comment]
> Rizz

Promote ur bot

**Reply by Noctophile** [reply]
it resolved by making super group . but here is new error bot is not replying on owner id but on others it will work fine.
![Screenshot from 2024-09-05 21-53-13](https://github.com/user-attachments/assets/75ce5b6c-bac3-462d-8343-86e319034d4f)

**Comment by anasty17** [comment]
Something wrong from ur side

**Reply by Noctophile** [reply]
will u add custom caption in it .please.................

**Reply by anasty17** [reply]
Open new issue
